devel/fnc/portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            fnc
+version         0.6
```

Comment:
In addition to `version`, we also request that ports also have a `revision` field:
```
version         0.6
revision        0
```

`revision` is incremented if changes are made to a port in a situation where its `version` isn't being updated.